CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 - [BREAKING] Replace `GetAccountProofs` with `GetAccountProof` in the public store API (#[1211](https://github.com/0xMiden/miden-node/pull/1211)).
 - Implemented storage map `DataStore` function ([#1226](https://github.com/0xMiden/miden-node/pull/1226)).
 - [BREAKING] Renamed `RemoteProverProxy` to `RemoteProverClient` ([#1236](https://github.com/0xMiden/miden-node/pull/1236)).
+- Updated `miden-base` dependency to latest version ([#1248](https://github.com/0xMiden/miden-node/pull/1248)).
```

Comment:
I'm not entirely sure that were adding this updates to the changelog, can you double check it please?